### PR TITLE
[GEARPUMP-295] Fix the failure of getting stalling tasks from clock s…

### PR DESCRIPTION
**Pull request by huafengw**
…ervice

Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the commit message is formatted like:
   `[GEARPUMP-<Jira issue #>] Meaningful description of pull request` 
 - [x] Make sure tests pass via `sbt clean test`.
 - [x] Make sure old documentation affected by the pull request has been updated and new documentation added for new functionality.

### CODE REVIEW

**Comment by manuzhang** [comment]
Let's wait for 0.8.3 release.

File: streaming/src/main/scala/org/apache/gearpump/streaming/appmaster/ClockService.scala [hunk 1]
```diff
@@ -261,7 +261,8 @@ class ClockService(
     case GetCheckpointClock =>
       sender ! CheckpointClock(minCheckpointClock)
 
-    case GetStallingTasks =>
+    // Please do not remove the `getStalling`
+    case getStalling: GetStallingTasks =>
```
Comment (manuzhang) [inline streaming/src/main/scala/org/apache/gearpump/streaming/appmaster/ClockService.scala]: If the field of `GetStallingTasks` is never used, we'd better turn it into a case object

**Comment by huafengw** [comment]
@manuzhang

**Comment by manuzhang** [comment]
+1

**Pull request closed without merging**